benchmark/gems/hearth.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Benchmark
+  module Gems
+    class Core < Benchmark::Gem
```

Comment:
I feel like hearth benchmark is pretty useless. It's also not aws/core ..

Reply:
Updated the name to be Hearth instead. I do think its useful to track the gem size and require time over time.